src/v/cluster/feature_manager.cc

```diff
@@ -247,7 +247,8 @@ ss::future<> feature_manager::do_maybe_update_active_version() {
     // This call in principle can be a network fetch, but in practice
     // we're only doing it immediately after cluster health has just
     // been updated, so do not expect it to go remote.
-    auto node_status_v = co_await _hm_frontend.local().get_nodes_status({});
+    auto node_status_v = co_await _hm_frontend.local().get_nodes_status(
```

Comment:
maybe we should use a timeout here ? Now it may lead to situation in which feature manage will stuck waiting on the result